libcsl_godot/src/ledger/transaction.rs

```diff
@@ -132,6 +136,71 @@ impl Datum {
     }
 }
 
+#[derive(GodotClass, Debug)]
+#[class(base=RefCounted, rename=Redeemer)]
+pub struct Redeemer {
+    pub redeemer: CSL::plutus::Redeemer,
+}
+
+#[godot_api]
+impl Redeemer {
+    #[func]
+    fn create(
+        tag: u64,
+        index: u64,
+        data: PackedByteArray,
+        ex_units_mem: u64,
+        ex_units_steps: u64,
+    ) -> Gd<Redeemer> {
+        let redeemer_tag: RedeemerTag = match tag {
+            0 => RedeemerTag::new_spend(),
+            1 => RedeemerTag::new_mint(),
+            2 => RedeemerTag::new_cert(),
+            3 => RedeemerTag::new_reward(),
+            _ => RedeemerTag::new_mint(),
```

Comment:
Is this a placeholder? I would expect for any tag outside of the range to produce an error.